src/main/scala/tech/cryptonomic/conseil/generic/chain/QueryProtocolPlatform.scala

```diff
@@ -0,0 +1,30 @@
+package tech.cryptonomic.conseil.generic.chain
+
+import tech.cryptonomic.conseil.generic.chain.QueryProtocolTypes.Query
+import tech.cryptonomic.conseil.tezos.ApiOperations
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/** Companion object providing default apply implementation */
+object QueryProtocolPlatform {
+  def apply(): QueryProtocolPlatform = new QueryProtocolPlatform(Map("tezos" -> ApiOperations))
```

Comment:
I would recommend we restructure this. The generic chain package should contain the 'contract' each specific implementation (e.g. Tezos or Ethereum) should implement. The specific logic should reside in each specific package, i.e. there should only be abstract behavior and no default behavior. Perhaps there should only be traits in this package and implementing classes / objects in the specific packages. What do you think?

Reply:
I think that was the whole point of the #69 ticket - I've implemented there logic which provided service implementation per (platform, network) pair. At this point the only difference was different database per service, but when there would come different implementation for lets say `ETH` we could just need to inject it. 

So at this point in time should I move the logic here from this other PR or just take care of it next after this one?

Reply:
Take care of it next. :smile: